### PR TITLE
Export utils as part of lib

### PR DESCRIPTION
**Pull request by NicolaasDieBaas**
lib/index.js' module.export should include utils.js

### CODE REVIEW

**Comment by NicolaasDieBaas** [comment]
@gsvr please review

File: lib/index.js [hunk 1]
```diff
@@ -2,10 +2,12 @@ var IdentityStore = require('./identity_store');
...
 var MessageSender = require('./message_sender');
+var utils = require('./utils.js');
 
 module.exports = {
     IdentityStore: IdentityStore,
     Hub: Hub,
     StageBasedMessaging: StageBasedMessaging,
-    MessageSender: MessageSender
+    MessageSender: MessageSender,
+    utils: utils
```
Comment (gsvr) [inline lib/index.js]: Capitalize for consistency?

**Comment by gsvr** [comment]
One comment, then :+1:

File: lib/index.js (hunk 1 above)
Comment (NicolaasDieBaas) [inline lib/index.js]: i would leave it as is, camelcase used for 'classes' and lowercase for collection of functions in util.js

Comment (gsvr) [inline lib/index.js]: Ok, note I've already :+1:ed it

**Pull request merged**